utils/file_compression.py

```diff
@@ -0,0 +1,40 @@
+import glob
+import zlib
+import zipfile
+import os
+
+def compress_files(regex="age_project*", out_file="age_project.zip"):
```

Comment:
please don't have default arguements

Reply:
Does this apply for everything or just the compress_files function? For now I left the others and only deleted this one for the re-review.